sphinx/application.py

```diff
@@ -117,8 +116,7 @@ def __init__(self, srcdir, confdir, outdir, doctreedir, buildername,
                  parallel=0):
         # type: (unicode, unicode, unicode, unicode, unicode, Dict, IO, IO, bool, bool, List[unicode], int, int) -> None  # NOQA
         self.verbosity = verbosity
-        self._extensions = {}                   # type: Dict[unicode, Any]
-        self._extension_metadata = {}           # type: Dict[unicode, Dict[unicode, Any]]
```

Comment:
I removed both `_extensions` and `_extension_metadata`. That might affect to the extensions refers them. But I determined to keep compatibility for them. Because they starts with underscore. so it is usually treated as private member in Python.
What do you think?